scripts/check-coverage.jsh

```diff
@@ -35,20 +35,20 @@ var moduleCoverageData = List.of(
   new ModuleCoverage("syson-application-configuration", 72.0),
   new ModuleCoverage("syson-common-view", 100.0),
   new ModuleCoverage("syson-diagram-common-view", 89.0),
-  new ModuleCoverage("syson-diagram-services", 80.0),
+  new ModuleCoverage("syson-diagram-services", 81.0),
   new ModuleCoverage("syson-direct-edit-grammar", 67.0),
   new ModuleCoverage("syson-form-services", 100.0),
-  new ModuleCoverage("syson-model-services", 88.0),
+  new ModuleCoverage("syson-model-services", 91.0),
   new ModuleCoverage("syson-representation-services", 100.0),
-  new ModuleCoverage("syson-services", 68.0),
+  new ModuleCoverage("syson-services", 64.0),
```

Comment:
No new code in this module, I only moved code into news modules. But the code moved was in average better tested that the remaining code